mlflow/models/rag_signatures.py

```diff
@@ -10,38 +10,38 @@
     Property,
     Schema,
 )
-from mlflow.utils.annotations import experimental
+from mlflow.utils.annotations import deprecated
 
 
 @dataclass
-@experimental
+@deprecated("mlflow.types.llm.ChatMessage")
 class Message:
     role: str = "user"  # "system", "user", or "assistant"
     content: str = "What is mlflow?"
 
 
 @dataclass
-@experimental
+@deprecated("mlflow.types.llm.ChatCompletionRequest")
 class ChatCompletionRequest:
     messages: list[Message] = field(default_factory=lambda: [Message()])
 
 
 @dataclass
-@experimental
+@deprecated("mlflow.types.llm.ChatCompletionRequest")
 class SplitChatMessagesRequest:
```

Comment:
Why is this replaced by ChatCompletionRequest? `query` field doesn't exist in ChatCompletionRequest

Reply:
We're deprecating this schema completely as we no longer plan to support this signature